crates/modelardb_server/src/storage/compressed_data_manager.rs

```diff
@@ -65,7 +66,7 @@ pub(super) struct CompressedDataManager {
     /// Channels used by the storage engine's threads to communicate.
     channels: Arc<Channels>,
     /// Management of metadata for saving compressed file metadata.
-    metadata_manager: Arc<MetadataManager>,
+    metadata_manager: Arc<TableMetadataManager<Sqlite>>,
```

Comment:
Maybe renamed to `table_metadata_manager` for consistency.

Reply:
Fixed.